apps/sms/test/marionette/mocks/mock_navigator_moz_mobile_message.js

```diff
@@ -3,6 +3,7 @@
 
 const Cu = Components.utils;
 Cu.import('resource://gre/modules/Services.jsm');
+Cu.import('resource://gre/modules/Promise.jsm');
```

Comment:
are you sure? I know some jsm that use promises without importing this... (for example https://dxr.mozilla.org/mozilla-central/source/dom/settings/SettingsRequestManager.jsm)